tests/wpt/tests/css/cssom-view/getBoundingClientRect-scroll.html

```diff
@@ -0,0 +1,91 @@
+<!DOCTYPE html>
+<title>getBoundingClientRect for a element inside scroll container</title>
+<link rel="author" title="Jo Steven Novaryo" href="mailto:steven.novaryo@gmail.com">
+<link rel="help" href="https://drafts.csswg.org/cssom-view/#dom-element-getboundingclientrect">
+<script src="https://wpt.live/resources/testharness.js"></script>
+<script src="https://wpt.live/resources/testharnessreport.js"></script>
+<html>
+  <head>
+    <style>
+      body {
+        margin: 0;
+      }
+      .scroll_container {
+        width: 300px;
+        height: 300px;
+        overflow: scroll;
+        scrollbar-width: none;
+      }
+      .dummy_overflowing_box {
+        width: 5000px;
+        height: 5000px;
+      }
+      .target_box {
+        width: 100px;
+        height: 100px;
+        background-color: green;
+      }
+      .position_absolute {
+        position: absolute;
+      }
+      .position_fixed {
+        position: fixed;
+      }
+      .absolute_containing_block {
+        position: relative;
+      }
+      .fixed_containing_block {
+        will-change: transform;
+      }
+    </style>
+  </head>
+  <body>
+    <div class="scroll_container fixed_containing_block">
+      <div class="scroll_container absolute_containing_block">
+        <div class="scroll_container">
+          <div class="target_box"></div>
+          <div class="dummy_overflowing_box"></div>
+        </div>
+        <div class="dummy_overflowing_box"></div>
+      </div>
+      <div class="dummy_overflowing_box"></div>
+    </div>
+    <div id="log" class="dummy_overflowing_box"></div>
+    <script>
+      setup(() => {
+        var offset_left = 1;
+        var offset_top = 2;
+        window.scrollTo(offset_left, offset_top);
+        for (scroll_container of document.getElementsByClassName('scroll_container')) {
+          offset_left *= 4;
+          offset_top *= 4;
+          scroll_container.scrollTo(offset_left, offset_top)
+        }
+      });
+      var target = document.querySelector('.target_box');
+      test(function() {
+        let staticRect = target.getBoundingClientRect();
+        assert_equals(staticRect.left, -85, 'static positioned target left');
+        assert_equals(staticRect.top, -170, 'static positioned target top');
+        assert_equals(staticRect.width, 100, 'static positioned target width');
+        assert_equals(staticRect.height, 100, 'static positioned target height');
+      }, "getBoundingClientRect for element inside scroll container");
+      test(function() {
+        target.style.position = 'absolute';
```

Comment:
Could you use different test elements? It's better to make test cases independent of each other, if possible.